src/test/java/core/basesyntax/service/validation/ValidatorImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.validation;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ValidatorImplTest {
+    private static String[] validData;
+    private static String[] invalidData;
+    private static String[] dataWithNegativeNumber;
+    private static Validator validator;
+    private boolean actual;
+
+    @BeforeClass
+    public static void beforeClass() {
+        validData = new String[]{"b", "banana", "1000"};
+        invalidData = new String[]{"apple", "10"};
+        dataWithNegativeNumber = new String[]{"b", "avocado", "-10"};
+        validator = new ValidatorImpl();
+    }
+
+    @Test
+    public void validate_correctData_ok() {
+        actual = validator.validate(validData);
+        assertTrue(actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void validate_incorrectDataWithNegativeNumber_NotOk() {
+        actual = validator.validate(dataWithNegativeNumber);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void validate_dataWithoutFirstElement_NotOk() {
+        actual = validator.validate(invalidData);
+    }
```

Comment:
What about other types of invalid data? For example {"b", "banana", "1000", "500"} or {"10", "10", "10"} ?